--- .Xresources
+++ .Xresources
@@ -4 +4 @@ Xft.autohint:               false
-Xft.dpi:                    123
+Xft.dpi:                    102
--- .bashrc
+++ .bashrc
@@ -51,0 +52,4 @@ cd() {
+diffasn() {
+  diff <(dumpasn1 -ip "$1") <(dumpasn1 -ip "$2")
+}
+
@@ -56,0 +61,8 @@ downiso() {
+dumpasn() {
+  less <(dumpasn1 -ip "$1")
+}
+
+lapo() {
+  "$BROWSER" "lapo.it/asn1js/#$(xxd -p $1 | paste -sd '')"
+}
+
@@ -71,0 +84,7 @@ man() {
+oid() {
+  for o in "$@" ; do
+    awk -F"[{}]" '/asn1/ {getline; print $2; exit}'                           \
+      <(curl -sN "http://oidref.com/$o")
+  done
+}
+
@@ -77,0 +97,5 @@ pacsize() {
+rfc() {
+  urxvtc -g 72x56 -e sh -c "i3-msg floating enable &&                         \
+    curl -s "ietf.org/rfc/rfc$1.txt" | sed '/\f/d; 1,3d' | less"
+}
+
--- .gitconfig
+++ .gitconfig
@@ -2 +2 @@
-    email = gzmbnn@gmail.com
+    email = gustavo.zambonin@grad.ufsc.br
--- .gitignore_global
+++ .gitignore_global
@@ -347,0 +348,52 @@ pyvenv.cfg
+
+# eclipse
+.metadata
+bin/
+tmp/
+*.tmp
+*.bak
+*.swp
+*~.nib
+local.properties
+.settings/
+.loadpath
+.recommenders
+
+# Eclipse Core
+.project
+
+# External tool builders
+.externalToolBuilders/
+
+# Locally stored "Eclipse launch configurations"
+*.launch
+
+# PyDev specific (Python IDE for Eclipse)
+*.pydevproject
+
+# CDT-specific (C/C++ Development Tooling)
+.cproject
+
+# JDT-specific (Eclipse Java Development Tools)
+.classpath
+
+# Java annotation processor (APT)
+.factorypath
+
+# PDT-specific (PHP Development Tools)
+.buildpath
+
+# sbteclipse plugin
+.target
+
+# Tern plugin
+.tern-project
+
+# TeXlipse plugin
+.texlipse
+
+# STS (Spring Tool Suite)
+.springBeans
+
+# Code Recommenders
+.recommenders/
--- .tmux.conf
+++ .tmux.conf
@@ -73 +73 @@ set -g status-right '\
-#[fg=colour244]#(amixer get Master -M | awk -f $HOME/.files/scripts/volume)\
+#[fg=colour244]#(amixer -c 1 get Master -M | awk -f ~/.files/scripts/volume)\
--- .wmconfig
+++ .wmconfig
@@ -1 +1 @@
-set $mod                    Mod1
+set $mod                    Mod4
@@ -24,10 +24,16 @@
-bindsym $mod+1              workspace α
-bindsym $mod+2              workspace β
-bindsym $mod+3              workspace γ
-bindsym $mod+4              workspace δ
-bindsym $mod+5              workspace ε
-bindsym $mod+6              workspace θ
-bindsym $mod+7              workspace λ
-bindsym $mod+8              workspace π
-bindsym $mod+9              workspace σ
-bindsym $mod+0              workspace ω
+workspace α                 output VGA-1
+workspace β                 output HDMI-1
+workspace γ                 output VGA-1
+workspace δ                 output HDMI-1
+workspace ε                 output VGA-1
+workspace θ                 output HDMI-1
+workspace λ                 output VGA-1
+workspace π                 output HDMI-1
+workspace σ                 output VGA-1
+workspace ω                 output HDMI-1
+
+bindsym $mod+1              workspace α, workspace β
+bindsym $mod+2              workspace γ, workspace δ
+bindsym $mod+3              workspace ε, workspace θ
+bindsym $mod+4              workspace λ, workspace π
+bindsym $mod+5              workspace σ, workspace ω
@@ -56,4 +62,4 @@
-bindsym mod5+F10            exec --no-startup-id amixer -q set Master toggle
-bindsym mod5+F11            exec --no-startup-id amixer -q sset Master 1%-
-bindsym mod5+F12            exec --no-startup-id amixer -q sset Master 1%+
-bindsym Print               exec --no-startup-id scrot
+bindsym XF86AudioMute           exec amixer -c 1 -q set Master toggle
+bindsym XF86AudioLowerVolume    exec amixer -c 1 -q sset Master 1%-
+bindsym XF86AudioRaiseVolume    exec amixer -c 1 -q sset Master 1%+
+bindsym --release Print         exec scrot
@@ -80 +86 @@
-exec --no-startup-id feh --bg-center $HOME/.wallpaper.jpg
+exec --no-startup-id feh --bg-fill --no-xinerama $HOME/.wallpaper.{jpg,png}
