--- .Xresources
+++ .Xresources
@@ -4 +4 @@ Xft.autohint:               false
-Xft.dpi:                    123
+Xft.dpi:                    102
--- .bash_profile
+++ .bash_profile
@@ -10,4 +10,2 @@ export PROMPT_COMMAND='history -a'
-export XKB_DEFAULT_LAYOUT=us
-export XKB_DEFAULT_MODEL=pc105
-export XKB_DEFAULT_VARIANT=intl
-export XKB_DEFAULT_OPTIONS=compose:rwin,ctrl:nocaps,terminate:ctrl_alt_bksp
+export XKB_DEFAULT_LAYOUT=br
+export XKB_DEFAULT_MODEL=abnt2
@@ -19,2 +17,2 @@ fi
-if [ -z "$DISPLAY" ] && [ "$(tty)" = /dev/tty1 ] ; then
-  exec sway -c "$HOME/.files/.swayconfig"
+if [ -z "$DISPLAY" ] && [ -n "$XDG_VTNR" ] && [ "$XDG_VTNR" -eq 1 ] ; then
+  exec startx /usr/bin/i3
--- .bashrc
+++ .bashrc
@@ -47,0 +48,4 @@ cd() {
+diffasn() {
+  diff <(dumpasn1 -ip "$1") <(dumpasn1 -ip "$2")
+}
+
@@ -52,0 +57,8 @@ downiso() {
+dumpasn() {
+  less <(dumpasn1 -ip "$1")
+}
+
+lapo() {
+  "$BROWSER" "lapo.it/asn1js/#$(xxd -p $1 | paste -sd '')"
+}
+
@@ -67,0 +80,7 @@ man() {
+oid() {
+  for o in "$@" ; do
+    awk -F"[{}]" '/asn1/ {getline; print $2; exit}'                           \
+      <(curl -sN "http://oidref.com/$o")
+  done
+}
+
@@ -73,0 +93,5 @@ pacsize() {
+rfc() {
+  urxvtc -g 72x56 -e sh -c "i3-msg floating enable &&                         \
+    curl -s "ietf.org/rfc/rfc$1.txt" | sed '/\f/d; 1,3d' | less"
+}
+
--- .gitconfig
+++ .gitconfig
@@ -2 +2 @@
-    email = gzmbnn@gmail.com
+    email = gustavo.zambonin@grad.ufsc.br
--- .gitignore_global
+++ .gitignore_global
@@ -347,0 +348,52 @@ pyvenv.cfg
+
+# eclipse
+.metadata
+bin/
+tmp/
+*.tmp
+*.bak
+*.swp
+*~.nib
+local.properties
+.settings/
+.loadpath
+.recommenders
+
+# Eclipse Core
+.project
+
+# External tool builders
+.externalToolBuilders/
+
+# Locally stored "Eclipse launch configurations"
+*.launch
+
+# PyDev specific (Python IDE for Eclipse)
+*.pydevproject
+
+# CDT-specific (C/C++ Development Tooling)
+.cproject
+
+# JDT-specific (Eclipse Java Development Tools)
+.classpath
+
+# Java annotation processor (APT)
+.factorypath
+
+# PDT-specific (PHP Development Tools)
+.buildpath
+
+# sbteclipse plugin
+.target
+
+# Tern plugin
+.tern-project
+
+# TeXlipse plugin
+.texlipse
+
+# STS (Spring Tool Suite)
+.springBeans
+
+# Code Recommenders
+.recommenders/
--- .swayconfig
+++ .i3config
@@ -6 +6 @@ font                        pango:DejaVu Sans Mono 8
-default_border              pixel
+for_window [class="^.*"]    border 1pixel
@@ -8 +8 @@ default_border              pixel
-bindsym $mod+dead_grave     exec rofi -show run -no-show-match
+bindsym $mod+apostrophe     exec rofi -show run -no-show-match
@@ -21,0 +23 @@ bindsym $mod+Return         exec urxvtc
+bindsym $mod+Tab            exec rofi -show window -no-show-match -lines 5
@@ -23,10 +25,16 @@ bindsym $mod+Return         exec urxvtc
-bindsym $mod+1              workspace α
-bindsym $mod+2              workspace β
-bindsym $mod+3              workspace γ
-bindsym $mod+4              workspace δ
-bindsym $mod+5              workspace ε
-bindsym $mod+6              workspace θ
-bindsym $mod+7              workspace λ
-bindsym $mod+8              workspace π
-bindsym $mod+9              workspace σ
-bindsym $mod+0              workspace ω
+workspace α                 output VGA-1
+workspace β                 output HDMI-2
+workspace γ                 output VGA-1
+workspace δ                 output HDMI-2
+workspace ε                 output VGA-1
+workspace θ                 output HDMI-2
+workspace λ                 output VGA-1
+workspace π                 output HDMI-2
+workspace σ                 output VGA-1
+workspace ω                 output HDMI-2
+
+bindsym $mod+1              workspace α, workspace β
+bindsym $mod+2              workspace γ, workspace δ
+bindsym $mod+3              workspace ε, workspace θ
+bindsym $mod+4              workspace λ, workspace π
+bindsym $mod+5              workspace σ, workspace ω
@@ -55,5 +63,5 @@ bindsym $mod+Shift+0        move container to workspace ω
-bindsym XF86AudioMute           exec amixer -q set Master toggle
-bindsym XF86AudioLowerVolume    exec amixer -q sset Master 1%-
-bindsym XF86AudioRaiseVolume    exec amixer -q sset Master 1%+
-bindsym --release Print         exec swaygrab
-bindsym --release $mod+Print    exec swaygrab -u
+bindsym XF86AudioMute           exec amixer -c 1 -q set Master toggle
+bindsym XF86AudioLowerVolume    exec amixer -c 1 -q sset Master 1%-
+bindsym XF86AudioRaiseVolume    exec amixer -c 1 -q sset Master 1%+
+bindsym --release Print         exec scrot
+bindsym --release $mod+Print    exec scrot -u
@@ -78,0 +87,2 @@ exec --no-startup-id urxvtd -q -o -f
+exec --no-startup-id compton -b --no-fading-openclose
+exec --no-startup-id feh --bg-fill --no-xinerama $HOME/.wallpaper.{jpg,png}
@@ -80,5 +90 @@ exec --no-startup-id xrdb -merge $HOME/.Xresources
-exec --no-startup-id \
-    xmodmap -e "keycode 66 = comma less comma less ccedilla Ccedilla ccedilla"
-exec --no-startup-id xmodmap -e "clear lock"
-
-output "*" background $HOME/.wallpaper.jpg stretch
+exec --no-startup-id setxkbmap br
--- .tmux.conf
+++ .tmux.conf
@@ -73 +73 @@ set -g status-right '\
-#[fg=colour244]#(amixer get Master -M | awk -f $HOME/.files/scripts/volume)\
+#[fg=colour244]#(amixer -c 1 get Master -M | awk -f ~/.files/scripts/volume)\
--- vconsole.conf
+++ vconsole.conf
@@ -1 +1 @@
-KEYMAP=us-acentos
+KEYMAP=br-abnt2
--- /dev/null
+++ .xinitrc
@@ -0,0 +1 @@
+exec /usr/bin/i3 -c "$HOME/.i3config"
--- suspend@.service
+++ suspend@.service
@@ -6,0 +7,2 @@ User=%I
+Type=forking
+Environment=DISPLAY=:0
--- scripts/lock
+++ scripts/lock
@@ -4 +4 @@ TMPBG="/tmp/screen.png"
-swaygrab "$TMPBG"
+scrot "$TMPBG"
@@ -6 +6 @@ convert "$TMPBG" -scale 50% -blur 0x5 -scale 200% "$TMPBG"
-swaylock -u -i "$TMPBG"
+i3lock -u -i "$TMPBG"
--- scripts/power
+++ scripts/power
@@ -3,2 +3,3 @@
-case $(printf "lock\nhalt\nwarm\nkill" | rofi -columns 4 -dmenu -p :) in
-  lock) "$HOME/.files/scripts/lock" ;;
+case $(printf "dark\nlock\nhalt\nwarm\nkill" | rofi -columns 5 -dmenu -p :) in
+  dark) xset dpms force off ;;
+  lock) "$HOME/.files/scripts/lock" && xset dpms force off ;;
