--- .Xresources
+++ .Xresources
@@ -4 +4 @@ Xft.autohint:               false
-Xft.dpi:                    123
+Xft.dpi:                    166
--- .swayconfig
+++ .swayconfig
@@ -54,2 +54,4 @@ bindsym $mod+Shift+0        move container to workspace ω

+bindsym XF86MonBrightnessDown   exec light -U 1%
+bindsym XF86MonBrightnessUp     exec light -A 1%
 bindsym XF86AudioMute           exec amixer -q set Master toggle
@@ -79,6 +81,9 @@ exec --no-startup-id urxvtd -q -o -f
 exec --no-startup-id xrdb -merge $HOME/.Xresources
-exec --no-startup-id \
-    xmodmap -e "keycode 66 = comma less comma less ccedilla Ccedilla ccedilla"
-exec --no-startup-id xmodmap -e "clear lock"
+exec --no-startup-id light -S 1

-output "*" background $HOME/.wallpaper.jpg stretch
+output "*" background $HOME/.wallpaper.png fit
+
+input 1739:30383:DLL075B:01_06CB:76AF_To {
+  dwt enabled
+  tap enabled
+}
--- .tmux.conf
+++ .tmux.conf
@@ -72,0 +73,3 @@ set -g status-right '\
+#[fg=colour244]#(awk -v interface="wlp58s0" -f \
+                $HOME/.files/scripts/wireless /proc/net/wireless) | \
+#[fg=colour244]#(acpi | awk -f $HOME/.files/scripts/battery) | \
@@ -74,0 +78 @@ set -g status-right '\
+set -g status-right-length 60
